src/Cryptol/Eval/FFI/GenHeader.hs

```diff
@@ -53,28 +54,31 @@ data ConvertResult
                      -- for both Cryptol parameter and return type cases.
 
 -- | Convert a Cryptol foreign declaration into a C function declaration.
-convertFun :: (Name, FFIFunType) -> GenHeaderM C.Decln
-convertFun (fName, FFIFunType {..}) = do
-  let tpIdent = fmap nameIdent . tpName
-  typeParams <- traverse convertTypeParam (pickNames (map tpIdent ffiTParams))
-  -- Name the input args in0, in1, etc
-  let inPrefixes =
-        case ffiArgTypes of
-          [_] -> ["in"]
-          _   -> ["in" ++ show @Integer i | i <- [0..]]
-  inParams <- convertMultiType In $ zip inPrefixes ffiArgTypes
-  (retType, outParams) <- convertType Out ffiRetType
-    <&> \case
-      Direct u  -> (u, [])
-      -- Name the output arg out
-      Params ps -> (C.TypeSpec C.Void, map (prefixParam "out") ps)
-  -- Avoid possible name collisions
-  let params = snd $ mapAccumL renameParam Set.empty $
-        typeParams ++ inParams ++ outParams
-      renameParam names (C.Param u name) =
-        (Set.insert name' names, C.Param u name')
-        where name' = until (`Set.notMember` names) (++ "_") name
-  pure $ C.FunDecln Nothing retType (unpackIdent $ nameIdent fName) params
+convertFun :: (Name, FFI) -> GenHeaderM C.Decln
+convertFun (fName, cc) =
+  case cc of
+    CallAbstract {} -> undefined "XXX"
```

Comment:
We should implement this `undefined`, as it currently causes `:generate-foreign-header` to crash when run on a file containing `foreign abstract` functions.